pkg/repository/http_watcher.go

```diff
@@ -120,12 +103,12 @@ func (c *chartmuseum) Start() error {
 		c.logger.Error(err, "Failed to update repository")
 		return err
 	}
+
```

Comment:
The `Poll` function will execute `RepoUpdate` once, so this function does not need to be executed here.

Reply:
RepoUpdate() is removed in the HTTPWatcher.Start()!